test/test-musicbrainz-image.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
 
-set -e -u
+set -e -o pipefail -u
```

Comment:
```suggestion
set -eou pipefail
```

Reply:
Thanks but it doesn't make options more readable, it doesn't follow `set` specification either. Actually, `-e` and `-u` might even be replaced with `-o <option-name>` which is more explicit.

Reply:
Being more explicit is not always good cause you need to read more to get the same context.